src/pages/index.js

```diff
@@ -51,9 +30,15 @@ const newCardModalUrl = document.querySelector("#modal-Url");
 
 //Card Queries
 const addNewCardModal = document.querySelector("#add-new-card");
-const addNewCardForm = document.querySelector("#add-card-form");
-const addNewCardSubmit = document.querySelector("#new-card-submit");
-const cardList = document.querySelector(".cards__list");
+
+//API EVENTS
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5f2b588a-250e-4922-b74a-306ecb82bfe6",
+    "Content-Type": "application/json",
+  },
+});
 
 //VALIDATION
 const config = {
```

Comment:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
* You haven't corrected something https://github.com/MrSurge87/se_project_aroundtheus/pull/1#discussion_r1387584155

Reply:
From the 8th sprint all constants (`initialCards` and `config`) should be in file `constants.js` in folder `utils` to clean up the main file.